service/service.go

```diff
@@ -137,6 +162,20 @@ func New(config Config) (*Service, error) {
 // Boot starts top level service implementation.
 func (s *Service) Boot() {
 	s.bootOnce.Do(func() {
-		go s.releaseController.Boot(context.Background())
+		ctx := context.Background()
+		backOffFactory := func() backoff.Interface { return backoff.NewMaxRetries(7, 1*time.Second) }
```

Comment:
Why this is a factory and not just a backoff value?

Reply:
Good point, I did a lazy copy paste here.

Simplified it. PTAL